ubuntuStation/views.py

```diff
@@ -289,3 +303,28 @@ def show_questions():
     #entries=[dict(questiontext=q.questiontext, questionurl=q.questionurl) for q in question]
     entries=question
     return render_template('show_questions.html', questions=entries)
+
+@app.route('/createuser', methods=['GET', 'POST'])
+@login_required
+def create_user():
+    form=UserForm(request.form).new()
+    if request.method == 'POST' and form.validate():
+        user=User(name=form.name.data, email=form.email.data)
+        try:
+            db_session.add(user)
+            db_session.commit()
+        except:
+            db_session.rollback()
+            raise
+        finally:
+            db_session.close()
+        return redirect(url_for('show_users'))
+    else:
+        print(form.errors)
+    return render_template('user_form.html', form=form)
+
+@app.route('/showusers', methods=['GET'])
```

Comment:
This should be mentioned in the original issue. I was expecting only the login page to have been changed. Others may not be aware for the same reason.